pages/users/getting-started/install-metamask.md

```diff
@@ -1 +1,64 @@
-# Install Metamask
\ No newline at end of file
+# Install MetaMask
+
+MetaMask is a browser extension and wallet for accessing decentralized applications (dApps) on EVM-compatible chains. It allows users to interact with blockchain networks directly from their browser, making it an essential tool for interacting with the XRPL EVM sidechain.
+
+## Attention
```

Comment:
Remove this section and all the references that say that this project is in development